File: ma_node.cpp
#include<ros/ros.h>
#include<std_msgs/String.h>

void chao_callback(std_msgs::String msg)
{
    ROS_INFO(msg.data.c_str());
    
}

void yao_callback(std_msgs::String msg)
{
    ROS_WARN(msg.data.c_str());
    
}

int main(int argc,char *argv[])
{
    setlocale(LC_ALL,"");
    ros::init(argc,argv,"ma_node");
    
    ros::NodeHandle nh;
    ros::Subscriber sub = nh.subscribe("kuai_shang_che_kai_hei_qun",10,chao_callback);
    ros::Subscriber sub_2 = nh.subscribe("ge_ge_dai_wo",10,yao_callback);

    while(ros::ok())
    {
        ros::spinOnce();
    }

    return 0;
}

File: ma_node.py
#!/usr/bin/env python3
#coding=utf-8

import rospy
from std_msgs.msg import String

def chao_callback(msg):
    rospy.loginfo(msg.data)

def yao_callback(msg):
    rospy.logwarn(msg.data)

if __name__=="__main__":
    rospy.init_node("ma_node")

    sub = rospy.Subscriber("kuai_shang_che_kai_hei_qun",String,chao_callback,queue_size=10)
    sub_2 = rospy.Subscriber("ge_ge_dai_wo",String,yao_callback,queue_size=10)
    
    rospy.spin()